docs/qlin_genetic/prescription/variants.md

```diff
@@ -65,18 +64,26 @@ Click on the cell under the note column ![note column header](note_column_header
 
 ### Removing a Note
 
-Click on the cell under the note column ![note column header](note_column_header.png), in the row of the table corresponding to the desired variant, remove all the content in the note editor, and save the change.
+Click on the **cell** under the note column ![note column header](note_column_header.png), in the row of the table corresponding to the desired variant, remove all the content in the note editor, and save the change.
 
 ### Filter by Notes
 
 The ![filter](filter_icon.png) icon located in the header of the note column ![note column header](note_column_header.png) allows you to filter the table based on the presence or absence of a note.
 
 ## Clinical Interpretation
 
-The **Clinical Interpretation** modal allows for the interpretation and classification of SNV sequence variations from a request. It is accessible via the **Interpret** button in [Detailed Information on the Occurrence of the SNV Variant](#detailed-information-on-the-occurrence-of-the-snv-variant).
+The **Clinical Interpretation** modal allows for the interpretation and classification of SNV sequence variations from a request.
+
+It is accessible via the **Interpret** button in [Detailed Information on the Occurrence of the SNV Variant](#detailed-information-on-the-occurrence-of-the-snv-variant) and from the Clinical Interpretation icon ![filled Clinical Interpretation](interpretation_filled_icon.png) in the SNV table.
 
 The form follows the same layout, but the classification uses different terminology for somatic and germline SNV sequence variations.
 
+The filled-in Clinical Interpretation icon ![filled Clinical Interpretation](interpretation_filled_icon.png) indicates the presence of a Clinical Interpretation on a variant.
```

Comment:
The filled-in Clinical
Qu'est-ce que c'est censé être?

Reply:
c'est l'icon plein (remplis)
https://www.merriam-webster.com/dictionary/filled%20in 
je vais enlever le tirret